status-EIPs.md

```diff
@@ -0,0 +1,211 @@
+# Status EIPs standards 
+
+> Version: 0.1 (Draft)
+>
+> Authors: Ricardo Guilherme Schmidt <ricardo3@status.im>
+
+## Abstract
+
+In this specification, we describe how Status relates with EIPs.
+
+## Table of Contents
+- [Status EIPs standards ](#status-client-specification)
+  - [Abstract](#abstract)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Components](#components)
+  - [Security Considerations](#security-considerations)
+  - [Design Rationale](#design-rationale)
+  - [Footnotes](#footnotes)
+  - [Acknowledgements](#acknowledgements)
+
+## Introduction
+
+Status should follow all standards as possible. Whenever a feature is needed, it should be first checked if there is a standard for that, if not, Status should propose a standard.
+
+### Support table
```

Comment:
A bit hard to horizontally scroll, I wonder if it makes sense to flip these axes around?

Reply:
Good idea. Just transposed the table. I used this https://www.tablesgenerator.com/markdown_tables